packages/default-theme/components/SwProductDetails.vue

```diff
@@ -131,6 +135,12 @@ export default {
           (this.product.translated && this.product.translated.description))
       )
     },
+    shippingFree() {
+      return this.product && this.product.shippingFree
+    },
+    productNumber() {
+      return this.product && this.product.productNumber || null
+    },
```

Comment:
would be nice to add these two to `helpers` package
* `getProductFreeShipping` returning `boolean`
* `getProductNumber` returning `string`

Reply:
getProductFreeShipping, getProductNumber helpers created